src/index.html

```diff
@@ -11,12 +11,103 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="ul">
+          <li class="nav-li">
+            <a
+              href="#Apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Samsung"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Smartphones"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Laptops&Computers"
+              class="nav-link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Gadgets"
+              class="nav-link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Tablets"
+              class="nav-link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Photo"
+              class="nav-link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav-li">
+            <a
+              href="#Video"
+              class="nav-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
Remember about correct indentation between parent and child elements. Each level of nesting, including text, should have a 2-space offset. Also, do not leave blank lines between parent and child elements.